src/Service/PaymentMethodsFilterService.php

```diff
@@ -92,6 +94,11 @@ function ($value) use ($pmHandlerIdentifier) {
                     if (empty($paymentMethodFoundInResponse)) {
                         $originalPaymentMethods->remove($paymentMethodEntity->getId());
                     }
+                    // Remove ApplePay PM if the browser is not Safari
+                } elseif ($pmCode == ApplePayPaymentMethodHandler::getPaymentMethodCode()) {
+                    if ($isSafari !== 1) {
+                        $originalPaymentMethods->remove($paymentMethodEntity->getId());
+                    }
```

Comment:
I would get rid of the extra `if` block since`$isSafari` is already defined at the top
```suggestion
                } elseif ($pmCode == ApplePayPaymentMethodHandler::getPaymentMethodCode() && $isSafari !== 1) {
                    $originalPaymentMethods->remove($paymentMethodEntity->getId());
```

Reply:
🙏